extras/ovms-operator/Makefile

```diff
@@ -14,95 +14,176 @@
 # limitations under the License.
 #
 
-# Current Operator version
-VERSION ?= 0.1.0
-# Default bundle image tag
-BUNDLE_IMG ?= controller-bundle:$(VERSION)
-# Options for 'bundle-build'
+# VERSION defines the project version for the bundle.
+# Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without changing the standard setup, you can:
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+VERSION ?= 0.0.1
```

Comment:
Before I saw version 0.1.0. Does this concern the same thing? If so perhaps we should have the same way of defining version (0.1.0 or 0.0.1). 
btw. shouldn't it be 0.0.2 with the new version?